packages/nextjs/src/client-boundary/uiComponents.tsx

```diff
@@ -50,17 +52,18 @@ export const CreateOrganization = (props: CreateOrganizationProps) => {
 // "The inferred type of 'OrganizationProfile' cannot be named without a reference to ..."
 export const OrganizationProfile: typeof BaseOrganizationProfile = Object.assign(
   (props: OrganizationProfileProps) => {
-    return <BaseOrganizationProfile {...useRoutingProps('OrganizationProfile', props)} />;
+    const path = usePathnameWithoutWithCatchAll();
+    return <BaseOrganizationProfile {...useRoutingProps('OrganizationProfile', props, { path })} />;
   },
   { ...BaseOrganizationProfile },
 );
 
 export const SignIn = (props: SignInProps) => {
-  const { signInUrl } = useClerkNextOptions();
```

Comment:
I don't think we should remove the `signInUrl` here, this is needed if we want to keep value coming from `<ClerkProvider/>` or env variable `NEXT_PUBLIC_CLERK_SIGN_IN_URL`.

Reply:
I don't think we need to respect this - the env variable is supposed to be used so the SignIn component (and the middleware) knows where SignUp is mounted and vice versa.

You want to render the component at the path its mounted, and respecting the global value as the path means that the component will only be able to be mounted to a single route